### DISCUSSION TITLE
Name alfabet.

### DISCUSSION THREAD

**Discussion by Alps28** [electricitymaps/electricitymaps-contrib#3097] [Ideas] [answered]
Hi when I see the list of country, and want to find Denmark.... Is it not so good if you call the 3 Danish place de for øst Denmark and West Denmark Bornholm Denmark.
IT is much more easy to finde some if you list them like 
Denmark. Øst
Denmark West
Denmark Bornholm.

It is the same thing for  Norway...1,2,3,4,5.
N for Norway
S for Sweden
G for Germany

**Comment by jarek** [comment, marked as answer]
The list of countries/regions is not sorted alphabetically, but by gCO2eq/kWh, so I'm not sure how much having the name begin with "Denmark" would help?

In any event, searching the list should be best for this, there is a search bar right above the list:
![search](https://user-images.githubusercontent.com/47415/114931286-0a03a180-9e04-11eb-9146-04f9be956527.png)
